types/underscore/index.d.ts

```diff
@@ -3807,14 +3807,14 @@ declare module _ {
         * @param key Property of the object.
         * @return Function which accept an object an returns the value of key in that object.
         **/
-        property(key: string): (object: Object) => any;
+        property(key: string | string[]): (object: Object) => any;
 
         /**
         * Returns a function that will itself return the value of a object key property.
         * @param key The object to get the property value from.
         * @return Function which accept a key property in `object` and returns its value.
         **/
-        propertyOf(object: Object): (key: string) => any;
+        propertyOf(object: Object): (key: string | string) => any;
```

Comment:
`string | string` ...?

Reply:
Misstype. Fixed, thanks.

Reply:
@weswigham 
May You check the code one more time, please?